src/screens/authentication/create_new_wallet/DoubleCheckItScreen/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useEffect, useState} from 'react';
+import {ScrollView, StyleSheet} from 'react-native';
+import {ScreenProps} from "navigation/ScreenProps";
+import CreateNewWalletRouter from 'navigation/CreateNewWalletNavigation/CreateNewWalletRouter';
+import _ from 'lodash';
+import {CLayout, CHeader} from "components";
+import {scale} from "device";
+import CButton2 from "components/CButton2";
+import {useNavigation} from "@react-navigation/native";
+import {CheckItem} from "screens/authentication/create_new_wallet/components";
+import {StackNavigationProp} from "@react-navigation/stack";
+
+const numberOfRandom = 8;
+
+// @ts-ignore
+const DoubleCheckItScreen: React.FC<ScreenProps<CreateNewWalletRouter.DOUBLE_CHECK_IT_SCREEN>> = ({route}) => {
```

Comment:
We have 2 doublecheck screens, Are they covered in this PR ? or another PR ?

Reply:
it's only 1 screen with 2 difference  style of next button.